sandbox/settings.py

```diff
@@ -151,7 +151,8 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
     MEDIA_ROOT = os.path.join(DATA_DIR, "media")
```

Comment:
weird because you are just undoing what you did in the previous commit. This is not really new and should be squashed with the previous commit?

Reply:
In the first step I moved the file name. In this step I removed it. It's not really undoing what I've done.